<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>885</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>651</width>
      <height>881</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="cameraopen">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>520</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera Open</string>
      </property>
     </widget>
     <widget class="QLabel" name="gray_video">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border : 1px solid red</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="graychange">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>570</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border : 1px solid red</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imageshow">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>570</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border : 1px solid red</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="origin_video">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 1px solid blue</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="imageopen">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>810</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>image show</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>301</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="vbox"/>
     </widget>
     <widget class="QPushButton" name="cameraclose">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>520</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Camera Close</string>
      </property>
     </widget>
     <widget class="QPushButton" name="graychangebtn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>810</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>gray image</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu5_1">
    <property name="title">
     <string>5장예제</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action5_1"/>
    <addaction name="action5_2"/>
   </widget>
   <addaction name="menu5_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action5_1">
   <property name="text">
    <string>action5_1</string>
   </property>
  </action>
  <action name="action5_2">
   <property name="text">
    <string>action5_2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>camera_open()</slot>
  <slot>camera_close()</slot>
  <slot>display_image()</slot>
  <slot>convert_to_gray()</slot>
 </slots>
</ui>
